weather-app/src/tests/components/ForecastSummary.test.js

```diff
@@ -0,0 +1,43 @@
+import React from "react";
+import { render } from "@testing-library/react";
+import ForecastSummary from "../../components/ForecastSummary";
+
+describe("ForecastSummary", () => {
+  const validProps = {
+    date: 1111111,
+    description: "Stub description",
+    icon: 800,
+    temperature: {
+      min: 12,
+      max: 22,
+    },
+  };
+  it("renders correctly", () => {
+    const { asFragment } = render(
+      <ForecastSummary
+        date={validProps.date}
+        description={validProps.description}
+        icon={validProps.icon}
+        temperature={validProps.temperature}
+        onSelect={validProps.onSelect}
+      />,
+    );
+    expect(asFragment()).toMatchSnapshot();
+  });
+  it("renders correct values for props", () => {
+    const { getByText, getByTestId } = render(
```

Comment:
Its a common convention to leave a clear line before the start of your assertion blocks and also as mentioned before in between the test blocks themselves too:
```
    );

    expect(asFragment()).toMatchSnapshot();
  });

  it("renders correct values for props", () => {
    const { getByText, getByTestId } = render(
```